patterns/media-instagram-grid.php

```diff
@@ -22,11 +22,11 @@
 			<div class="wp-block-cover__inner-container">
 				<!-- wp:group {"style":{"dimensions":{"minHeight":"100%"},"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"center","justifyContent":"center"}} -->
 				<div class="wp-block-group" style="min-height:100%"><!-- wp:heading {"fontSize":"large"} -->
-					<h2 class="wp-block-heading has-large-font-size">Instagram</h2>
+					<h2 class="wp-block-heading has-large-font-size"><?php esc_html_e( 'Instagram', 'twentytwentyfive' ); ?></h2>
 					<!-- /wp:heading -->
 
 					<!-- wp:paragraph {"align":"center","fontSize":"medium"} -->
-					<p class="has-text-align-center has-medium-font-size">@<a href="#">example</a></p>
+					<p class="has-text-align-center has-medium-font-size"><?php esc_html_e( '@', 'twentytwentyfive' ); ?><a href="#"><?php esc_html_e( 'example', 'twentytwentyfive' ); ?></a></p>
```

Comment:
Can we combine `@` and `example` into one string?

Reply:
[This might be updated again later,](https://github.com/WordPress/twentytwentyfive/issues/468) but combining them it is the best alternative for now.